### DISCUSSION TITLE
Apply library to my already done project

### DISCUSSION THREAD

**Discussion by chato9595** [khoih-prog/ESP_WiFiManager#99] [Q&A]
Hello, thank you very much for this contribution because apparently it is just what I need. I come from using tzapu's Wifi Manager but it has the problem that it does not show the assigned IP after connecting to my router.
I honestly don't understand how to replace what you had already done with your library. I need an access point to be created and after storing my router credentials  show the assigned IP in the configuration portal and the credentials are saved for further connections.

**Comment by khoih-prog** [comment]
>  I need an access point to be created and after storing my router credentials show the assigned IP in the configuration portal and the credentials are saved for further connections

All features are there to be used. You have to test using some examples to figure out.

Good Luck,
